src/inputs/Table/Table.js

```diff
@@ -168,21 +192,18 @@ export const Table = (props) => {
           buildErrorsCountLabel={buildErrorsPanelTitle}
         />
       )}
-      <div data-cy="grid" id="grid-container" style={dimensions} className={agTheme}>
+      <div data-cy="grid" id="grid-container" className={agTheme}>
         {isReady && (
-          <AgGridReact
-            ref={gridRef}
-            undoRedoCellEditing={true}
-            rowDragManaged={true}
-            suppressDragLeaveHidesColumns={true}
-            allowDragFromColumnsToolPanel={true}
-            columnDefs={formattedColumns}
-            defaultColDef={defaultColDef}
-            columnTypes={columnTypes}
-            rowData={rows}
-            context={context}
-            stopEditingWhenCellsLoseFocus={true}
-          />
+          <>
+            <TableFilterRow isFullscreen={isFullscreen} toggleFullscreen={toggleFullscreen} />
+            <Box sx={dimensions}>{AgGridData}</Box>
+            <Dialog fullScreen open={isFullscreen} onClose={toggleFullscreen} className={agTheme}>
+              <Box p={4} sx={{ height: '100%' }}>
```

Comment:
Instead of creating a `Box` with a padding, please use a `<DialogContent>` that already includes the right amount of padding without any extra CSS required.

Reply:
Done